app/routes.py

```diff
@@ -0,0 +1,291 @@
+    if request.method == "POST":
+        request_body = request.get_json()
+        input_check = validate_video_input(request_body)
+        if input_check['error']: 
+            return make_response({"details":input_check['details']}), input_check['status_code']
+        else:    
+            new_video = Video(title=request_body['title'], release_date=request_body['release_date'], total_inventory=request_body['total_inventory'])
+            db.session.add(new_video)
+            db.session.commit()
+            return make_response(new_video.to_dict()), 201
+    elif request.method == "GET":
+        video_list = Video.query.all()
+        response = []
+        for my_video in video_list:
+            response.append(my_video.to_dict())
+        
+        return jsonify(response), 200
+
+
+@videos_bp.route("/<video_id>", methods=["GET", "PUT", "DELETE"], strict_slashes=False)
+def get_video(video_id):
+    try:
+        video_id = int(video_id)
+    except ValueError:
+        return "Error, please provide the integer id of the video", 400
+    my_video = Video.query.get(video_id)
+    if request.method == "GET":
+        if my_video is None:
+            return {"message": "Video " + str(video_id) + " was not found"}, 404
+        else:
+            return make_response(my_video.to_dict()), 200
+
+    elif request.method == "PUT":
+        request_body = request.get_json()
+        if my_video is None:
+            return {"message": "Video " + str(video_id) + " was not found"}, 404
+        else:
+            input_check = validate_video_input(request_body)
+            if input_check['error']: 
+                return make_response({"details":input_check['details']}), input_check['status_code']
+            else:  
+                my_video.title = request_body["title"]
+                my_video.release_date = request_body["release_date"]
+                my_video.total_inventory =request_body ["total_inventory"]
+                db.session.commit()        
+                return make_response(my_video.to_dict()), 200
+
+
+    elif request.method == "DELETE":
+        if my_video is None:
+            return {"message": "Video 1 was not found"}, 404
+        else:        
+            db.session.delete(my_video)
+            db.session.commit()
+            return make_response({"id":video_id}), 200
+
+#CUSTOMER ROUTES
+
+@customer_bp.route("",methods =["POST","GET"])
+def handle_customers():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "name" not in request_body:
+            return ({
+                "details": "Request body must include name."
+            }),400
+        elif "phone" not in request_body:
+            return ({
+                "details": "Request body must include phone."
+            }),400
+        elif "postal_code" not in request_body:
+            return ({
+                "details": "Request body must include postal_code."
+            }),400
```

Comment:
I just want to note how this validation section makes the function VERY long.  Breaking it out with a helper function or a decorator would be ideal.